syntax = "proto3";

package sl.sbc.message;

enum Command
{
    SHOW_STATUS = 0;
    SHOW_LINES = 1;
    INSERT_DATA = 2;
}

enum Return_code
{
    SUCCESS = 0;
    USER_NOT_FOUND = 1;
    PASSWORD_INCORRECT = 2;
    USER_DATA_LIMIT_REACHED = 3;
}

message Message
{
    string username = 1;
    string password = 2;
    string data = 3;
    Command com = 4;
    Return_code ret_c = 5;
}